### ISSUE TITLE
Lets Vote - Nishant Mittal

### ISSUE THREAD

**Issue by nishantwrp** [nishantwrp/nishantwrp-website#55]
https://www.nishantwrp.com/posts/lets-vote/ 

 An online platform to conduct elections built using Django and Azure Blockchain Services.

**Comment by ARUNEGIMONK** [comment]
<h1><i>i don't know what my level is how can i know that <i></h1>

**Comment by nishantwrp** [comment]
Didn't get your question. Can you please elaborate?

**Comment by ARUNEGIMONK** [comment]
Thanks for your reply. Can you please tell me even after studying daily I
am still a beginner

**Comment by nishantwrp** [comment]
I don't think this type of conversation should be done in the comment section. Feel me to ping me on my [email](mailto:admin@nishantwrp.com)
